docs/getting_started/scaleway.md

```diff
@@ -0,0 +1,59 @@
+# Getting Started with kops on Scaleway
+
+**WARNING**: Scaleway support on kOps is currently in **alpha**, which means that it is in the early stages of development and subject to change, please use with caution.
+
+## Requirements
+
+* [kops version >= 1.18 installed](../install.md)
+* [kubectl installed](../install.md)
+* [Setup your SSH key](https://www.scaleway.com/en/docs/configure-new-ssh-key/) : id_rsa and id_ed25519 keys are supported
+* [Scaleway credentials](https://www.scaleway.com/en/docs/generate-api-keys/) : you will need at least an access key, a secret key and a project ID.
+* [S3 bucket and its credentials](https://www.scaleway.com/en/docs/storage/object/quickstart/) : the bucket's credentials may differ from the one used for provisioning the resources needed by the cluster. If you use a Scaleway bucket, you will need to prefix the bucket's name with `scw://` in the `KOPS_STATE_STORE` environment variable. For more information about buckets, see [here](../state.md)
+
+### Optional
+* If you wish to use your own domain name for your cluster, you'll need a [Scaleway managed domain](https://www.scaleway.com/en/docs/network/domains-and-dns/quickstart). If not, you can always use the [gossip DNS](https://kops.sigs.k8s.io/gossip/) by naming your cluster with the suffix `.k8s.local`.
+
+## Environment Variables
+
+It is important to set the following [environment variables](https://github.com/scaleway/scaleway-sdk-go/blob/master/scw/README.md):
+```bash
+export SCW_ACCESS_KEY="my-access-key"
+export SCW_SECRET_KEY="my-secret-key"
+export SCW_DEFAULT_PROJECT_ID="my-project-id"
```

Comment:
Can this not go in the cluster spec and `kops create cluster` flag, like it does for GCP?

Reply:
Scaleway APIs use these environment variables to create the client. We can also use profiles from a config file but we have to keep in mind that not all our clients have one.

Reply:
Could the kOps Scaleway integration code deal with that?

Reply:
I see two possibles ways:
Option 1: Kops uses the scaleway configuration file already used for scaleway-cli and terraform. The advantage of this method is that it offers more comfort for scaleway users that already use the scaleway dev tools ecosystem. The disadvantage is that it multiplies potential files where to look for this.

Option 2: We only use the kops.yaml configuration file. Advantage: people that are using Kops are only maintaining one configuration file. Would the kops team be open to fetching secrets from different locations? We can use the `viper` library to bring both environment variables and configuration file-defined variables.